tests/integration/src/test/scala/mesosphere/marathon/integration/LeaderIntegrationTest.scala

```diff
@@ -245,8 +245,6 @@ class BackupRestoreIntegrationTest extends LeaderIntegrationTest {
 
       val tmpBackupFile = File.createTempFile("marathon", "BackupRestoreIntegrationTest")
 
-      waitForSSEConnect()
```

Comment:
Hmm, why did you remove this line? These were put in to prevent flakes related to missed events

Reply:
Is it because the test support code already waited for an SSE connection event?